# Fixes:
# /usr/bin/ld: gupnp-dlna-info.o: undefined reference to symbol 'gst_discoverer_audio_info_get_sample_rate'
# gupnp-dlna uses automake 1.10.3, so autogen.sh is required propagate
# Makefile.am changes
diff -urN gupnp-dlna-0.5.0.orig/autogen.sh gupnp-dlna-0.5.0/autogen.sh
--- gupnp-dlna-0.5.0.orig/autogen.sh	1970-01-01 09:30:00.000000000 +0930
+++ gupnp-dlna-0.5.0/autogen.sh	2011-01-14 20:12:52.138829403 +1030
@@ -0,0 +1,5 @@
+#! /bin/sh
+gtkdocize --flavour no-tmpl || exit 1
+ACLOCAL="${ACLOCAL-aclocal} $ACLOCAL_FLAGS" autoreconf -v --install || exit 1
+glib-gettextize --force --copy || exit 1
+./configure --enable-debug "$@"
diff -urN gupnp-dlna-0.5.0.orig/tools/Makefile.am gupnp-dlna-0.5.0/tools/Makefile.am
--- gupnp-dlna-0.5.0.orig/tools/Makefile.am	2011-01-08 00:42:42.000000000 +1030
+++ gupnp-dlna-0.5.0/tools/Makefile.am	2011-01-14 20:12:37.483829403 +1030
@@ -1,7 +1,8 @@
 bin_PROGRAMS = gupnp-dlna-info gupnp-dlna-ls-profiles
 
 AM_CFLAGS = -I$(top_srcdir) -I$(top_srcdir)/gst-convenience/gst-libs $(GST_CFLAGS)
-LIBS = $(GST_LIBS) \
+LIBS = $(GST_PBU_LIBS) \
+       $(GST_LIBS) \
        $(top_builddir)/libgupnp-dlna/libgupnp-dlna-1.0.la
 
 gupnp_dlna_ls_profiles_LDADD = \
